easybuild/easyconfigs/a/alleleCount/alleleCount-4.2.1-GCC-11.3.0.eb

```diff
@@ -0,0 +1,50 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+# 
+# Author: Jonas Demeulemeester
+# The Francis Crick Institute, London, UK
+##
+
+easyblock = 'CmdCp'
+
+name = 'alleleCount'
+version = '4.2.1'
+
+homepage = 'http://cancerit.github.io/alleleCount/'
+description = """ The alleleCount package primarily exists to prevent code duplication
+between some other projects, specifically AscatNGS and Battenberg. As of v4 the perl 
+code wraps the C implementation of allele counting code for BAM/CRAM processing. """
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/cancerit/%(name)s/archive/v%(version)s']
+sources = ['v%(version)s.tar.gz']
+checksums = ['dbec2d47fb4fad8ead72f55c4b6e3c02b216cbed285c05617b62e9a38c32e400']
+
+cmds_map = [('.*', "./setup.sh . ${EBROOTPERL}/lib/perl5")]
+
+# HTSlib 1.7 is built, used and removed again in the provided setup script
+dependencies = [
+    ('Perl', '5.34.1'),
+    ('HTSlib', '1.15.1'),
+    ('libdeflate', '1.10'),
```

Comment:
@ItIsI-Orient does it definitely no longer need `cURL`?

Reply:
Since we are using `HTSlib` and `libdeflate` locally, this package gets installed by the script `opt-build-local.sh`, which is called in the setup.sh script. None of those scripts use `cURL` in the version 4.2.1 (they were both using it in the version 4.0.0), so I would say that it is no longer a hard requirement.
It would be required in case we would like the script to download those dependencies.

Reply:
Fine, thanks for the explanation.